The simplex tests fail when numpy and scipy are installed

CC @smichr 
```console
pytest sympy/solvers/tests/test_simplex.py
========================================================= test session starts ==========================================================
platform darwin -- Python 3.12.2, pytest-8.0.2, pluggy-1.4.0
architecture: 64-bit
cache:        yes
ground types: python

rootdir: /Users/enojb/work/dev/sympy
configfile: pyproject.toml
plugins: timeout-2.2.0, hypothesis-6.98.17, doctestplus-1.2.0, xdist-3.5.0, split-0.8.2
collected 4 items

sympy/solvers/tests/test_simplex.py F...                                                                                         [100%]

=============================================================== FAILURES ===============================================================
_______________________________________________________________ test_lp ________________________________________________________________

    def test_lp():
        r1 = y + 2*z <= 3
        r2 = -x - 3*z <= -2
        r3 = 2*x + y + 7*z <= 5
        constraints = [r1, r2, r3, x >= 0, y >= 0, z >= 0]
        objective = -x - y - 5 * z
        variables = [x, y, z]
        ans = optimum, argmax = lp(max, objective, constraints)
        assert ans == lpmax(objective, constraints)
        if scipy is not None and np is not None:
>           scipy_res = get_results_with_scipy(
                objective, constraints, variables)

sympy/solvers/tests/test_simplex.py:56:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

objective = -x - y - 5*z, constraints = [y + 2*z <= 3, -x - 3*z <= -2, 2*x + y + 7*z <= 5, x >= 0, y >= 0, z >= 0]
variables = [x, y, z]

    def get_results_with_scipy(objective, constraints, variables):
        if scipy is not None and np is not None:
>           from sympy.solvers.inequalities import _np
E           ImportError: cannot import name '_np' from 'sympy.solvers.inequalities' (/Users/enojb/work/dev/sympy/sympy/solvers/inequalities.py)

sympy/solvers/tests/test_simplex.py:27: ImportError
                                                            DO *NOT* COMMIT!
======================================================= short test summary info ========================================================
FAILED sympy/solvers/tests/test_simplex.py::test_lp - ImportError: cannot import name '_np' from 'sympy.solvers.inequalities' (/Users/enojb/work/dev/sympy/sympy/solvers/inequalities.py)
===================================================== 1 failed, 3 passed in 0.67s ======================================================
```
There is no `_np` function. The test should be fixed or removed.

--- a/sympy/solvers/tests/test_simplex.py
+++ b/sympy/solvers/tests/test_simplex.py
@@ -1,12 +1,12 @@
 from sympy.core.numbers import Rational
-from sympy.core.relational import Relational, Eq, Ne
+from sympy.core.relational import Eq, Ne
 from sympy.core.symbol import symbols
 from sympy.core.sympify import sympify
 from sympy.core.singleton import S
 from sympy.core.random import random, choice
 from sympy.functions.elementary.miscellaneous import sqrt
 from sympy.ntheory.generate import randprime
-from sympy.matrices.dense import Matrix, eye
+from sympy.matrices.dense import Matrix
 from sympy.solvers.solveset import linear_eq_to_matrix
 from sympy.solvers.simplex import (_lp as lp, _primal_dual,
     UnboundedLPError, InfeasibleLPError, lpmin, lpmax,
@@ -29,13 +29,8 @@ def test_lp():
     r3 = 2*x + y + 7*z <= 5
     constraints = [r1, r2, r3, x >= 0, y >= 0, z >= 0]
     objective = -x - y - 5 * z
-    variables = [x, y, z]
     ans = optimum, argmax = lp(max, objective, constraints)
     assert ans == lpmax(objective, constraints)
-    if scipy is not None and np is not None:
-        scipy_res = get_results_with_scipy(
-            objective, constraints, variables)
-        assert optimum.evalf() == sympify(-scipy_res.fun)
     assert objective.subs(argmax) == optimum
     for constr in constraints:
         assert constr.subs(argmax) == True
@@ -45,13 +40,8 @@ def test_lp():
     r3 = 2*x + y - 7*z <= -5
     constraints = [r1, r2, r3, x >= 0, y >= 0, z >= 0]
     objective = -x - y - 5*z
-    variables = [x, y, z]
     ans = optimum, argmax = lp(max, objective, constraints)
     assert ans == lpmax(objective, constraints)
-    if scipy is not None and np is not None:
-        scipy_res = get_results_with_scipy(
-            objective, constraints, variables)
-        assert optimum.evalf() == sympify(-scipy_res.fun)
     assert objective.subs(argmax) == optimum
     for constr in constraints:
         assert constr.subs(argmax) == True
@@ -62,13 +52,8 @@ def test_lp():
     constraints = [r1, r2, r3, x >= 0, y >= 0, z >= 0]
     const = 2
     objective = -x-y-5*z+const # has constant term
-    variables = [x, y, z]
     ans = optimum, argmax = lp(max, objective, constraints)
     assert ans == lpmax(objective, constraints)
-    if scipy is not None and np is not None:
-        scipy_res = get_results_with_scipy(
-            objective-const, constraints, variables)
-        assert optimum.evalf() == (sympify(-scipy_res.fun)+const)
     assert objective.subs(argmax) == optimum
     for constr in constraints:
         assert constr.subs(argmax) == True
@@ -92,12 +77,7 @@ def test_lp():
     r3 = 2*x + y - 7*z <= 10
     constraints = [r1, r2, r3] + [i >= 0 for i in (x, y, z)]
     objective = -x-y-5*z
-    variables = [x, y, z]
     optimum, argmax = lp(max, objective, constraints)
-    if scipy is not None and np is not None:
-        scipy_res = get_results_with_scipy(
-            objective, constraints, variables)
-        assert optimum.evalf() == sympify(-scipy_res.fun)
     assert objective.subs(argmax) == optimum
     for constr in constraints:
         assert constr.subs(argmax) == True
@@ -133,74 +113,14 @@ def test_lp():
         objective = sum(rand() * x for x in variables)
         return objective, constraints, variables
 
-    # testing random problems
-    if scipy is not None and np is not None:
-        for _ in range(50):
-            objective, constraints, variables = make_random_problem()
-            # in case c auto simplifies to True or False
-            constraints = [c for c in constraints
-                if isinstance(c, Relational)]
-            if len(constraints) == 0:
-                continue
-
-            # check lp maximization
-            # scipy minimizes, so negative objective for it
-            scipy_res = get_results_with_scipy(
-                -objective, constraints, variables)
-            if scipy_res.status == 0:
-                optimum, argmax = lp(max, objective, constraints)
-                scipy_op = -scipy_res.fun  # negative gives actual max
-                assert abs(optimum.evalf() - scipy_op) < .1**10
-                assert objective.subs(argmax) == optimum
-                for constr in constraints:
-                    assert constr.subs(argmax) == True
-            elif scipy_res.status == 2:
-                # scipy: problem is infeasible
-                raises(InfeasibleLPError,
-                       lambda: lp(max, objective, constraints))
-            elif scipy_res.status == 3:
-                # scipy: problem is unbounded
-                raises(UnboundedLPError,
-                       lambda: lp(max, objective, constraints))
-            else:
-                # scipy: iteration limit or numerical difficulties
-                pass
-
-            # check lp minimization
-            scipy_res = get_results_with_scipy(
-                objective, constraints, variables)
-            if scipy_res.status == 0:
-                optimum, argmax = lp(min, objective, constraints)
-                scipy_op = scipy_res.fun
-                assert abs(optimum.evalf() - scipy_op) < .1**10
-                assert objective.subs(argmax) == optimum
-                for constr in constraints:
-                    assert constr.subs(argmax) == True
-            elif scipy_res.status == 2:
-                # scipy: problem is infeasible
-                raises(InfeasibleLPError,
-                       lambda: lp(max, objective, constraints))
-            elif scipy_res.status == 3:
-                # scipy: problem is unbounded
-                raises(UnboundedLPError,
-                       lambda: lp(max, objective, constraints))
-            else:
-                # scipy: iteration limit or numerical difficulties
-                pass
-
     # equality
     r1 = Eq(x, y)
     r2 = Eq(y, z)
     r3 = z <= 3
     constraints = [r1, r2, r3]
     objective = x
-    variables = [x, y, z]
     ans = optimum, argmax = lp(max, objective, constraints)
     assert ans == lpmax(objective, constraints)
-    if scipy is not None and np is not None:
-        scipy_res = get_results_with_scipy(
-            objective, constraints, variables)
-        assert optimum.evalf() == sympify(-scipy_res.fun)
     assert objective.subs(argmax) == optimum
     for constr in constraints:
         assert constr.subs(argmax) == True

--- a/sympy/solvers/tests/test_simplex.py
+++ b/sympy/solvers/tests/test_simplex.py
@@ -22,26 +22,6 @@ from sympy.abc import x, y, z
 np = import_module("numpy")
 scipy = import_module("scipy")
 
-def get_results_with_scipy(objective, constraints, variables):
-    if scipy is not None and np is not None:
-        from sympy.solvers.inequalities import _np
-        nonpos, rep, xx = _np(constraints, [])
-        assert not rep  # only testing nonneg variables
-        C, _D = linear_eq_to_matrix(objective, *variables)
-        A, B = linear_eq_to_matrix(nonpos, *variables)
-        assert _D[0] == 0  # scipy only deals with D = 0
-
-
-
-        A_sci = Matrix([[A], [-eye(len(variables))]])
-        B_sci = Matrix([[B], [Matrix([0] * len(variables))]])
-        C_sci = C
-        A_sci = np.array(A_sci.tolist())
-        B_sci = np.array(B_sci.tolist())
-        C_sci = np.array(C_sci.tolist())
-        res = scipy.optimize.linprog(C_sci, A_ub=A_sci, b_ub=B_sci)
-        return res
-
 
 def test_lp():
     r1 = y + 2*z <= 3

